rpc/block_svc_routines.c

```diff
@@ -2738,6 +2774,17 @@ getSoTgArraysForAllVolume(struct soTgObj *obj, blockGenConfigCli *blk,
           goto out;
         }
 
+        if (!info->prio_path[0]) {
+            /* default as the load balancing is enabled */
+            list = blockMetaInfoToServerParse(info);
```

Comment:
@lxbsz Should we defend for NULL return here ?

Reply:
Just copied this code from somewhere which didn't defend this:
`
list = blockServerParse(blk->block_hosts);
 `
But defend for NULL here will make sense. Will update this.

Thanks.

Reply:
Feel free to fix the other place as well :-)

Reply:
sure.